src/org/parosproxy/paros/extension/option/OptionsConnectionPanel.java

```diff
@@ -854,7 +858,19 @@ private JCheckBox getCheckBoxSingleCookeRequestHeader() {
         return checkBoxSingleCookieRequestHeader;
     }
 
-    private ProxyExcludedDomainsMultipleOptionsPanel getProxyExcludedDomainsPanel() {
+    public JCheckBox getCheckBoxHttpStateEnabled() {
+        if (checkBoxHttpStateEnabled == null) {
+            checkBoxHttpStateEnabled =
+                    new JCheckBox(Constant.messages.getString("conn.options.httpStateEnabled"));
+        }
+        return checkBoxHttpStateEnabled;
+    }
+
+	public void setCheckBoxHttpStateEnabled(JCheckBox checkBoxHttpStateEnabled) {

```

Comment:
I know it's a setter but do we really need it? What's the use case? If a new check box is set the UI will stop working as expected (the one previously added to the UI is not being replaced).